front/components/agent_builder/triggers/ScheduleEditionModal.tsx

```diff
@@ -42,12 +42,21 @@ function formatTimezone(timezone: string): string {
   return `${city} (${timezone})`;
 }
 
+function isErrorWithMessage(err: unknown): err is { error: { message: string } } {
+  return (
+    typeof err === "object" &&
+    err !== null &&
+    "error" in err &&
+    typeof (err as any).error === "object" &&
+    (err as any).error !== null &&
```

Comment:
do we need the `as any` ? I'd expect the `"error" in err` to be enough

Reply:
removed